fluent-bundle/src/resolver/scope.rs

```diff
@@ -8,28 +8,28 @@ use std::borrow::Borrow;
 use std::fmt;
 
 /// State for a single `ResolveValue::to_value` call.
-pub struct Scope<'scope, 'errors, R, M> {
+pub struct Scope<'bundle, 'ast, 'args, 'errors, R, M> {
```

Comment:
Thought: This is a little gross with the proliferation of the lifetimes, but it seems like a reasonable approach to solve the issue. It at least makes it more explicit.

Reply:
Thought: Reading through it a few times, it at least makes things very explicit for each reference.